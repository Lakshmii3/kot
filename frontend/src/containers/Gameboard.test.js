import React from 'react';
import { shallow } from 'enzyme';

import GameboardLayout from './Gameboard';

describe('Verify gameboard layout template', () => {
  it('Gameboard should render correctly', () => {
    const errorPage = shallow(<GameboardLayout />);
    expect(errorPage).toMatchSnapshot();
  });
})