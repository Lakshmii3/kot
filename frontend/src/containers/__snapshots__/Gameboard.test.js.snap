// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Verify gameboard layout template Gameboard should render correctly 1`] = `
<div>
  <Adapter>
    <Adapter />
    <Adapter>
      <Sider>
        <DiceRoller />
      </Sider>
      <Adapter>
        <GameConsole />
      </Adapter>
      <Sider />
    </Adapter>
    <Adapter />
  </Adapter>
</div>
`;
